tests/services/run_devices/pacbio/test_pacbio_sequencing_runs_service.py

```
@@ -0,0 +1,23 @@
+from cg.server.endpoints.sequencing_run.dtos import PacbioSequencingRunsResponse
+from cg.services.run_devices.pacbio.sequencing_runs_service import PacbioSequencingRunsService
+from cg.store.models import PacbioSequencingRun
+
```

add test when none present should raise the error

---

Done